modules/checklist/assets/js/components/checklist-card-content.js

```diff
@@ -1,8 +1,24 @@
 import { Button, Card, CardActions, CardContent, CardMedia, Link, Typography } from '@elementor/ui';
 import PropTypes from 'prop-types';
 
-const ChecklistCardContent = ( props ) => {
-	const { description, learn_more_url: learnMoreUrl, learn_more_text: learnMoreText, cta_text: CTA, id, image_src: imageSrc } = props.step.config;
+const ChecklistCardContent = ( { step } ) => {
+	const {
+		id,
+		description,
+		learn_more_url: learnMoreUrl,
+		learn_more_text: learnMoreText,
+		image_src: imageSrc,
+		is_locked: isLocked,
+		promotion_url: promotionUrl,
+		cta_url: ctaUrl,
+	} = step.config;
+
+	const ctaText = isLocked ? __( 'Upgrade Now', 'elementor-pro' ) : step.config.cta_text,
```

Comment:
Do you need to `import { __ } from '@wordpress/i18n';`

Reply:
I thought about that, it didn't offer my the import and it seemed to work, do you want me to import it just to be on the safe side?

Reply:
Import it rather